### PR TITLE
Make image and url absolute path for meta tags

### PR DESCRIPTION
**Pull request by DelGall**

### CODE REVIEW

File: marble/templates/base.html [hunk 1]
```diff
@@ -20,18 +20,18 @@
     <!-- Twitter Meta Tags -->
     <meta name="twitter:card" content="summary_large_image">
     <meta property="twitter:domain" content="tworock.io">
-    <meta property="twitter:url" content="{{ domain }}{{ request.get_full_path }}">
+    <meta property="twitter:url" content="https://tworock.io/">
```
Comment (PeterGrand) [inline marble/templates/base.html]:
```suggestion
    <meta property="twitter:url" content="{{ request.scheme }}://{{ request.get_host }}{{ request.get_full_path }}">
```

File: marble/templates/base.html [hunk 2]
```diff
@@ -20,18 +20,18 @@
...
     <meta name="twitter:title" content="{% if page.seo_title %}{{ page.seo_title }}{% else %}{{ page.title }}  - {{ current_site.site_name }}{% endif %}">
     <meta name="twitter:description" content="{{ page.search_description }}">
-    <meta name="twitter:image" content="{% static 'images/two-rock-software-social.jpg' %}">
+    <meta name="twitter:image" content="https://tworock.io/static/images/two-rock-software-social.jpg">
 
 
     <!-- Facebook Meta Tags -->
-    <meta property="og:url" content="{{ domain }}{{ request.get_full_path }}">
+    <meta property="og:url" content="https://tworock.io/">
```
Comment (PeterGrand) [inline marble/templates/base.html]:
```suggestion
    <meta property="og:url" content="{{ request.scheme }}://{{ request.get_host }}{{ request.get_full_path }}">
```

File: marble/templates/base.html [hunk 3]
```diff
@@ -20,18 +20,18 @@
     <!-- Twitter Meta Tags -->
     <meta name="twitter:card" content="summary_large_image">
     <meta property="twitter:domain" content="tworock.io">
-    <meta property="twitter:url" content="{{ domain }}{{ request.get_full_path }}">
+    <meta property="twitter:url" content="https://tworock.io/">
     <meta name="twitter:title" content="{% if page.seo_title %}{{ page.seo_title }}{% else %}{{ page.title }}  - {{ current_site.site_name }}{% endif %}">
     <meta name="twitter:description" content="{{ page.search_description }}">
-    <meta name="twitter:image" content="{% static 'images/two-rock-software-social.jpg' %}">
+    <meta name="twitter:image" content="https://tworock.io/static/images/two-rock-software-social.jpg">
```
Comment (PeterGrand) [inline marble/templates/base.html]:
```suggestion
    <meta name="twitter:image" content="{{ request.scheme }}://{{ request.get_host }}{{ request.get_full_path }}{% static 'images/two-rock-software-social.jpg' %}">
```

File: marble/templates/base.html [hunk 4]
```diff
@@ -20,18 +20,18 @@
...
 
 
     <!-- Facebook Meta Tags -->
-    <meta property="og:url" content="{{ domain }}{{ request.get_full_path }}">
+    <meta property="og:url" content="https://tworock.io/">
     <meta property="og:type" content="website">
     <meta property="og:title" content="{% if page.seo_title %}{{ page.seo_title }}{% else %}{{ page.title }}  - {{ current_site.site_name }}{% endif %}">
     <meta property="og:description" content="{{ page.search_description }}">
-    <meta property="og:image" content="{% static 'images/two-rock-software-social.jpg' %}">
+    <meta property="og:image" content="https://tworock.io/static/images/two-rock-software-social.jpg">
```
Comment (PeterGrand) [inline marble/templates/base.html]:
```suggestion
    <meta property="og:image" content="{{ request.scheme }}://{{ request.get_host }}{{ request.get_full_path }}{% static 'images/two-rock-software-social.jpg' %}">
```

**Review by PeterGrand — CHANGES_REQUESTED** [review]
It would be ideal to populate these tags dynamically instead of hard coding the urls. So I've added suggestions for that.

**Review by DelGall — COMMENTED** [review]
Updated meta tags.

**Pull request merged**